src/implementations/BigFloat.jl

```diff
@@ -371,17 +389,7 @@ function buffered_operate_to!(
     x::AbstractVector{F},
     y::AbstractVector{F},
 ) where {F<:BigFloat}
-    local set! = function (out::F, in::F)
-        ccall(
-            (:mpfr_set, :libmpfr),
-            Int32,
-            (Ref{BigFloat}, Ref{BigFloat}, Base.MPFR.MPFRRoundingMode),
-            out,
-            in,
-            Base.MPFR.ROUNDING_MODE[],
-        )
-        return nothing
-    end
+    set! = (o, i) -> operate_to!(o, copy, i)
```

Comment:
I think you need to keep the `local ` to avoid the performance issue of closures

Reply:
Besides, the tests would have caught any extra allocations.